cinema/serializers.py

```diff
@@ -1 +1,74 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+class GenreListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "name"
```

Comment:
The `fields` attribute should be a tuple or list, not a string. Change `fields = "name"` to `fields = ("name",)` or `fields = ["name"]`.